<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlNumInputModule</class>
 <widget class="QWidget" name="CtrlNumInputModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QPushButton
{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;
border:1px solid transparent;
border-radius:5px;
}</string>
  </property>
  <widget class="QPushButton" name="Num1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:',微软雅黑'; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Numc">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:white;
font:20px,&quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
